controllers/model/modelversion_controller.go

```diff
@@ -136,15 +136,17 @@ func (r *ModelVersionReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 
 	modelVersionStatus := modelVersion.Status.DeepCopy()
 
-	// create kaniko pod to build container image\
-	// take the preceding 5 chars for image versionId
-	versionId := string(modelVersion.UID[:5])
+	// create kaniko pod to build container image, take the preceding 5 chars for image versionId
+	versionId := "v" + string(modelVersion.UID[:5])
```

Comment:
prepend 'v' may be not necessary, because the uid itself might use random letters.